core/metacling/src/TCling.cxx

```diff
@@ -1167,6 +1146,7 @@ TCling::TCling(const char *name, const char *title, const char* const argv[])
    fClingCallbacks(0), fAutoLoadCallBack(0),
    fTransactionCount(0), fHeaderParsingOnDemand(true), fIsAutoParsingSuspended(kFALSE)
 {
+   fFirstRun = true;
```

Comment:
First run of what? Please specifically state what flips this boolean. I'd suggest `fParsedSharedLibSymbols` or similar.